test/e2e/uninstall/uninstall_test.go

```diff
@@ -40,35 +42,61 @@ func deleteCRD(crdName string) {
 // We deliberately call out to kubectl, rather than constructing the client-go
 // objects, so that we can verfiy the local deploy yamls are correct.
 func teardownMayastor() {
-	// The correct sequence for a reusable  cluster is
-	// Delete all pods in the default namespace
-	// Delete all pvcs
-	// Delete all mayastor pools
-	// Then uninstall mayastor
-	podsDeleted, podCount := common.DeleteAllPods()
-	pvcsDeleted, pvcsFound := common.DeleteAllVolumeResources()
+	var podsDeleted bool
+	var pvcsDeleted bool
+	var podCount int
+	var pvcsFound bool
+
+	logf.Log.Info("Settings:", "cleanup", cleanup)
+	if !cleanup {
+		podCount, podsListOK := common.CheckNoTestPods()
+		if !podsListOK {
+			logf.Log.Info("WARNING: failed to list Pods")
```

Comment:
What about removing "WARNING" and change the Info to an Error Log?

Reply:
dohh I meant to write a Warning Log

Reply:
There isn't a logf.Log.Warn. logf.Log.Error is escalated to a failure immediately.
This code is executed when not  cleaning up, so if enumerating PVCs, PVs and MSVs fails the test run is a failed.
I've chopped up the code checking for various resources into smaller bits so that we can handle failure in a more
nuanced manner it the need should arise in the future.

Reply:
Oh that's sad :(